.github/workflows/build-and-test.yml

```diff
@@ -31,8 +31,15 @@ jobs:
     name: Codespell
     uses: Lombiq/GitHub-Actions/.github/workflows/codespell.yml@dev
 
+  lint:
+    name: Lint Script and Style
```

Comment:
We need MD linting too in this project.

Reply:
Added, here is an example: https://github.com/OrchardCMS/OrchardCore.Commerce/actions/runs/14821047779/job/41608428919?pr=577#step:3:237

Reply:
I've added MD linting, only allow it for the whole repo because giving more configurability here feels like an inconvenience and a bad idea. [Here is an example error.](https://github.com/OrchardCMS/OrchardCore.Commerce/actions/runs/14821047779/job/41608613936?pr=577#step:3:257)

![image](https://github.com/user-attachments/assets/71929a2b-995a-4c9e-99ab-f1780d513202)